### ISSUE TITLE
Unavailable Amazon RDS as for the building new server

### ISSUE THREAD

**Issue by ttksm** [treasure-data/digdag#1360]
I got the error message when I built a new server using digdag `v0.9.41` with Amazon RDS.
```
Caused by: org.skife.jdbi.v2.exceptions.UnableToExecuteStatementException: org.postgresql.util.PSQLException: ERROR: permission denied to create extension "uuid-ossp"
  Hint: Must be superuser to create this extension. [statement:"CREATE EXTENSION IF NOT EXISTS "uuid-ossp"", located:"CREATE EXTENSION IF NOT EXISTS "uuid-ossp"", rewritten:"CREATE EXTENSION IF NOT EXISTS "uuid-ossp"", arguments:{ positional:{}, named:{}, finder:[]}]
```
<details>
<summary>more logs</summary>

```
Caused by: org.skife.jdbi.v2.exceptions.UnableToExecuteStatementException: org.postgresql.util.PSQLException: ERROR: permission denied to create extension "uuid-ossp"
  Hint: Must be superuser to create this extension. [statement:"CREATE EXTENSION IF NOT EXISTS "uuid-ossp"", located:"CREATE EXTENSION IF NOT EXISTS "uuid-ossp"", rewritten:"CREATE EXTENSION IF NOT EXISTS "uuid-ossp"", arguments:{ positional:{}, named:{}, finder:[]}]
        at org.skife.jdbi.v2.SQLStatement.internalExecute(SQLStatement.java:1338)
        at org.skife.jdbi.v2.Update.execute(Update.java:56)
        at org.skife.jdbi.v2.BasicHandle.update(BasicHandle.java:298)
        at io.digdag.core.database.migrate.Migration_20151204221156_CreateTables.migrate(Migration_20151204221156_CreateTables.java:17)
        at io.digdag.core.database.DatabaseMigrator.applyMigration(DatabaseMigrator.java:239)
        at io.digdag.core.database.DatabaseMigrator.lambda$applyMigrationIfNotDone$1(DatabaseMigrator.java:146)
        at org.skife.jdbi.v2.tweak.transactions.LocalTransactionHandler.inTransaction(LocalTransactionHandler.java:183)
        at org.skife.jdbi.v2.BasicHandle.inTransaction(BasicHandle.java:331)
        at io.digdag.core.database.DatabaseMigrator.applyMigrationIfNotDone(DatabaseMigrator.java:139)
        at io.digdag.core.database.DatabaseMigrator.migrate(DatabaseMigrator.java:96)
        at io.digdag.core.database.DatabaseModule$AutoMigrator.migrate(DatabaseModule.java:62)
        ... 46 more
Caused by: org.postgresql.util.PSQLException: ERROR: permission denied to create extension "uuid-ossp"
  Hint: Must be superuser to create this extension.
        at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2458)
        at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2158)
        at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:291)
        at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:432)
        at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:358)
        at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:171)
        at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:160)
        at com.zaxxer.hikari.pool.ProxyPreparedStatement.execute(ProxyPreparedStatement.java:44)
        at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.execute(HikariProxyPreparedStatement.java)
        at org.skife.jdbi.v2.SQLStatement.internalExecute(SQLStatement.java:1327)
        ... 56 more
```
</details>

However, Amazon RDS does not permit to use or create superuser.
As for the enabling extensions of PostgreSQL, parameter group `rds.extensions` is provided instead of this constrict.
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts

`uuid-ossp` module is actually available by the parameter group.
```sql
postgres=> select * from pg_available_extensions where name = 'uuid-ossp';
-[ RECORD 1 ]-----+------------------------------------------------
name              | uuid-ossp
default_version   | 1.1
installed_version |
comment           | generate universally unique identifiers (UUIDs)
```

**Comment by hiroyuki-sato** [comment]
Hello, @ttksm 

Many users use digdag on Amazon RDS.
Have you ever tried it with administrator privileges?
(For investigating the problem)

**Comment by ttksm** [comment]
Thank you for your reply, @hiroyuki-sato 

I have tried building a new digdag server use a master role on Amazon RDS following your advice.
Consequently, it was succeeded.

To obtain administrator privileges on Amazon RDS, roles actually have to be inherited the role `rds_superuser`.
The role, I had been using firstly, did not have this administrator privileges.
So, I have granted `rds_superuser` to the role, that resolved this problem.

Thank you so much!
